tests/input_files/beam3r_line2_backweuler_browndyn_largesys.dat

```diff
@@ -61,8 +61,7 @@ AZTOL                  1.0E-3
 AZCONV                 AZ_r0
 AZITER                 1000
 AZSUB                  50
-IFPACKGFILL            1
-IFPACKOVERLAP          1
+IFPACK_XML_FILE                 xml/ifpack.xml
```

Comment:
This places the xml file into the top level directory of all xml files for solvers/preconditioners. There is an existing folder structure. @maxfirmbach, should we sort it into the folder structure? If yes, where? Into a new directory?

Reply:
@dharinib98 @mayrmt We need to figure out a place, not sure where it fits best, in theory we could just create a folder called `preconditioner` and put all preconditioner related .xml files there with an updated naming scheme. So `xml/preconditioner/ifpack.xml` would work for me right now.